CalCli.API/AlarmActions.cs

```diff
@@ -7,6 +7,7 @@ namespace CalCli.API
 {
     public enum AlarmActions
     {
+        NONE,
```

Comment:
To my knowledge, the behavior defined in [CalDav RFC](https://tools.ietf.org/html/rfc4791#section-8.6), has nothing to tell about the NONE action. With this change, we are implementing something more tolerant than the API itself.

Can you please provide documentation on how recurring alarms or events are stored or the reason to parse `NONE` as a valid action?

Reply:
Zimbra returns the following where there is a reminder set to 5 minutes before the event.

    BEGIN:VALARM
    ACTION:NONE
    TRIGGER;VALUE=DATE-TIME:19760401T005545Z
    DESCRIPTION:Reminder
    X-WR-ALARMUID:1C414420-2E9C-47D0-9084-9E110714D53A
    END:VALARM
    BEGIN:VALARM
    ACTION:DISPLAY
    TRIGGER:-PT5M
    DESCRIPTION:Reminder
    X-WR-ALARMUID:4DFA4853-A1FA-46AD-8BF1-C706821FB79F
    END:VALARM

I have no idea why Zimbra does this just that it does.